resources/views/theme-list.blade.php

```diff
@@ -0,0 +1,138 @@
+@extends('layouts.app')
```

Comment:
this file and theme-overview.blade.php have very similar code, i recommend to split them in smaller reusable chunks/components